static/templates/gear_menu.hbs

```diff
@@ -0,0 +1,129 @@
+<ul class="nav" role="navigation">
+    <li class="dropdown actual-dropdown-menu" id="gear-menu">
+        <a id="settings-dropdown" href="#" role="button" class="dropdown-toggle" data-target="nada" data-toggle="dropdown" title="{{t 'Menu' }} (g)">
+            <i class="fa fa-cog settings-dropdown-cog" aria-hidden="true"></i>
+        </a>
+        <ul class="dropdown-menu" role="menu" aria-labelledby="settings-dropdown">
+            {{!-- It is quite ingrained in our frontend code that your Home
+            view is a Bootstrap nav tab, even though we don't show the tab
```

Comment:
Yep, We do use the Bootstrap nav tab for our Home View (`#message_feed_container`).

https://github.com/zulip/zulip/blob/dd90083ed7f3dc5d1e6e17ac44f5e9ad20213508/static/js/hashchange.js#L132-L134

https://github.com/zulip/zulip/blob/dd90083ed7f3dc5d1e6e17ac44f5e9ad20213508/static/js/ui_util.js#L6-L7

Reply:
OK.  We will probably want to remove that as part of migrating away from using bootstrap, but that should definitely not be in this code move PR. :)